### PR TITLE
Add translations for 16.2

### PR DESCRIPTION
**Pull request by def-**
Translated German while at it
Screenshot 2022-06-14 at 20 31 27
Screenshot 2022-06-14 at 20 31 08

## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by C0D3D3V** [comment]
If we still want to move ddnet character display info into ddnet character, I would suggest to let https://github.com/ddnet/ddnet/pull/5210 be a blocking PR, I will work on this now, and have an Idea how to fix the compatibility problem with expanding Net Objects, (I hope I get this finished in the next two days). 
Alternatively,  I could move the flags to  ddnet character (that is done fast), and we then just drop the support for the HUD for 16.2 after we moved all fields to ddnet character

**Comment by C0D3D3V** [comment]
Ups I somehow thought this is a Version Release PR xD I was lost

**Comment by C0D3D3V** [comment]
But while we're on the subject of translating new terms.
How about presenting the freeze bar to the user in the menu as "Defrost process bar" instead of "Freeze bar". I had only used the term in the code, but actually it has more to do with thawing than freezing.

**Comment by def-** [comment]
I think it's always been called freeze and unfreeze in the community.

File: data/languages/german.txt [hunk 1]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Bisher keine Antwort vom Server.
+
+Download community skins
+== Community-Skins runterladen
+
+Device
+== Gerät
+
+Status
+== Statu
```
Comment (murpii) [inline data/languages/german.txt:1416]: s*

File: data/languages/german.txt [hunk 2]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Spielerposition zeigen
+
+Show player speed
+== Spielergeschwindigkeit zeigen
+
+Show player target angle
+== Zielwinkel des Spielers zeigen
+
+Ingame mouse sens.
+== Maussens. im Spiel
```
[the remark this answers is not in the document]
Comment (def-) [reply data/languages/german.txt:1443]: Wouldn't find. Empf. seems less clear than sens. to me.

[the remark this answers is not in the document]
Comment (heinrich5991) [reply data/languages/german.txt:1443]: What would 'sens.' mean in German?

[the remark this answers is not in the document]
Comment (murpii) [reply data/languages/german.txt:1443]: > What would 'sens.' mean in German?

Sensivität

[the remark this answers is not in the document]
Comment (def-) [reply data/languages/german.txt:1443]: Sensibilität? :D

[the remark this answers is not in the document]
Comment (murpii) [reply data/languages/german.txt:1443]: Both work I guess :P Google is giving me mixed feelings

**Comment by Jupeyy** [comment]
lol

**Comment by Jupeyy** [comment]
well nvm i create new pr

File: data/languages/german.txt [hunk 3]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== UDP scheint gefiltert zu werden.
+
+UDP and TCP IP addresses seem to be different. Try disabling VPN, proxy or network accelerators.
+== UDP- und TCP-IP-Adressen scheinen unterschiedlich zu sein. Versuch VPN, Proxy oder Netzwerkbeschleuniger abzuschalten.
+
+No answer from server yet.
+== Bisher keine Antwort vom Server.
+
+Download community skins
+== Community-Skins runterladen
```
Comment (AssassinTee) [inline data/languages/german.txt:1410]: **he**runterladen

File: data/languages/german.txt [hunk 4]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Ziel-Bind
+
+Mouse
+== Maus
+
+Use DDRace HUD
+== DDRace-HUD nutzen
+
+Show dummy actions
+== Dummy-Aktionen zeigen
```
Comment (AssassinTee) [inline data/languages/german.txt:1428]: vielleicht auch **anzeigen** aber naja ist okay

File: data/languages/german.txt [hunk 5]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Mause-Empfind. im Menü
+
+Enable controller
+== Controller aktivieren
+
+Controller %d: %s
+== Controller %d: %s
+
+Click to cycle through all available controllers.
+== Klicken um durch die verfügbaren Controller zu wechseln.
```
Comment (AssassinTee) [inline data/languages/german.txt:1455]: `Klicke,` um durch die verfügbaren Controller zu wechseln.

File: data/languages/german.txt [hunk 6]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Absolut
+
+Ingame controller mode
+== Spiel-Controller-Modus
+
+Ingame controller sens.
+== Controller-Empfind. im Spiel
+
+UI controller sens.
+== Controller-Empfind. Im Menü
```
Comment (AssassinTee) [inline data/languages/german.txt:1472]: **im**

File: data/languages/german.txt [hunk 7]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+
+Controller jitter tolerance
+== Jitter-Toleranz des Controllers
+
+No controller found. Plug in a controller and restart the game.
+== Kein Controller gefunden. Schließ einen Joystick an und starte das Spiel neu.
```
Comment (AssassinTee) [inline data/languages/german.txt:1478]: **Keinen** ? not sure about that one

**Review by AssassinTee — COMMENTED** [review]
German grammar _people_

File: data/languages/german.txt [hunk 8]
```diff
@@ -1393,3 +1384,101 @@ Team %d
...
+== Winkel:
+
+Trying to determine UDP connectivity...
+== Versuche UDP-Verbindungsfähigkeit festzustellen...
+
+UDP seems to be filtered.
+== UDP scheint gefiltert zu werden.
+
+UDP and TCP IP addresses seem to be different. Try disabling VPN, proxy or network accelerators.
+== UDP- und TCP-IP-Adressen scheinen unterschiedlich zu sein. Versuch VPN, Proxy oder Netzwerkbeschleuniger abzuschalten.
```
[the remark this answers is not in the document]
Comment (Jupeyy) [reply data/languages/german.txt:1404]: both are legal actually, i'd also prefer versuche, but the other is shorter xd

File: data/languages/german.txt (hunk 3 above)
Comment (Jupeyy) [reply data/languages/german.txt:1410]: that is "Umgangssprachlich", so kinda ok too

File: data/languages/german.txt (hunk 4 above)
Comment (Jupeyy) [reply data/languages/german.txt:1428]: I'd say anzeigen is better in this case indeed, I'll fix it in my pr

File: data/languages/german.txt (hunk 7 above)
Comment (Jupeyy) [reply data/languages/german.txt:1478]: Der Controller, so yes

**Pull request merged**